trio_model.cc

```diff
@@ -381,22 +381,39 @@ void TrioModel::SequencingProbabilityMat() {
   double max_element = read_dependent_data_.sequencing_probability_mat.maxCoeff();
   read_dependent_data_.max_elements.push_back(max_element);
 
+//  cout << read_dependent_data_.sequencing_probability_mat.array() << endl << endl;
+//  cout << likelihood_read_dependent_data_.sequencing_probability_mat.array() << endl << endl;
+
+  // Log likelihood uses same data matrixes without the constant max_element.
+  likelihood_read_dependent_data_.sequencing_probability_mat = exp(
+          read_dependent_data_.sequencing_probability_mat.array()
+  );
+
+//  cout << read_dependent_data_.sequencing_probability_mat.array() << endl << endl;
+//  cout << likelihood_read_dependent_data_.sequencing_probability_mat.array() << endl << endl;
+
   // Calculates sequencing_probability_mat and splits into individual child
   // mother, and father vectors.
   read_dependent_data_.sequencing_probability_mat = exp(
-    read_dependent_data_.sequencing_probability_mat.array() - max_element
+    read_dependent_data_.sequencing_probability_mat.array()
+            - max_element
   );
   read_dependent_data_.child_somatic_probability = read_dependent_data_.sequencing_probability_mat.row(0);
   read_dependent_data_.mother_somatic_probability = read_dependent_data_.sequencing_probability_mat.row(1);
   read_dependent_data_.father_somatic_probability = read_dependent_data_.sequencing_probability_mat.row(2);
 
-  // Log likelihood uses same data matrixes without the constant max_element.
-  likelihood_read_dependent_data_.sequencing_probability_mat = exp(
-    read_dependent_data_.sequencing_probability_mat.array()
-  );
-  likelihood_read_dependent_data_.child_somatic_probability = read_dependent_data_.sequencing_probability_mat.row(0);
-  likelihood_read_dependent_data_.mother_somatic_probability = read_dependent_data_.sequencing_probability_mat.row(1);
-  likelihood_read_dependent_data_.father_somatic_probability = read_dependent_data_.sequencing_probability_mat.row(2);
+//  cout << read_dependent_data_.sequencing_probability_mat.array() << endl << endl;
+//  cout << likelihood_read_dependent_data_.sequencing_probability_mat.array() << endl << endl;
+
+// Note: BUG: likelihood_read_dependent_data_ got these values from read_dependent_data_
```

Comment:
I found this one too but it didn't seem to change my results (still decreasing log likelihood).